### PR TITLE
feat: adds Supabase to database management

### PR DESCRIPTION
**Pull request by awalias**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [ x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [ x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x ] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x ] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x ] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x ] Comments and unused optional fields have been removed.
- [x ] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x ] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [ x] Any software project you are adding to the list is actively maintained.
- [ x] Any software project you are adding was first released more than 4 months ago.
- [ x] Any software project you are adding has working installation instructions.
- [ x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [ x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Review by kokomo123 — COMMENTED** [review]
Considering it's still in public beta and not "general release" I would say it could be added to the list, but im not sure. I'm open to other people's thoughts on this one. It looks relatively mature at this stage though.

File: software/supabase.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Supabase
+website_url: https://supabase.com
+description: Firebase alternative built on Postgres. Includes auth, realtime, file store, and a vector db.
+licenses:
+  - Apache-2.0
+platforms:
+  - Docker
+tags:
+  - Database Management
```
Comment (nodiscc) [inline software/supabase.yml:9]:
```suggestion
tags:
  - Software Development - FaaS & Serverless
```

We already list a similar solution: https://awesome-selfhosted.net/#nhost (_Firebase Alternative with GraphQL. Get a database and backend configured and ready in minutes._) which is tagged `Software Development - FaaS & Serverless`. I think `Database Management` is traditionally used for web frontends/graphical DB management tools in the context of the list.

File: software/supabase.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+source_code_url: https://github.com/supabase/supabase
+stargazers_count: 57973
+updated_at: '2023-10-16'
+archived: false
```
Comment (nodiscc) [inline software/supabase.yml:13]:
```suggestion
```

These fields will be populated automatically, let's remove these.

**Review by nodiscc — CHANGES_REQUESTED** [review]
> Considering it's still in public beta and not "general release" I would say it could be added to the list

Since the documentation is clear about the intended use-case (_Public Beta: Stable enough for most non-enterprise use-cases_), and that there have been regular releases since 2021, I don't think the beta status is a problem.

**Review by kokomo123 — APPROVED** [review]
After the suggested changes have been made, I'm happy to approve this PR. Thanks for your contribution! 💯

**Pull request closed without merging**